packages/core/strapi/lib/Strapi.js

```diff
@@ -416,7 +413,11 @@ class Strapi {
 
     await this.runLifecyclesFunctions(LIFECYCLES.BOOTSTRAP);
 
-    this.cron.start();
+    if (strapi.config.get('server.cron.enabled', true)) {
+      const cronTasks = this.config.get('server.cron.tasks', {});
+      this.cron.add(cronTasks, 'user');
+      this.cron.start('user');
```

Comment:
I don't understand the need for this specific start in the if?

I would conditionally register but always start the service that will just start all registered jobs and if there are none do nothing.